go/kbfs/tlfhandle/handle.go

```diff
@@ -48,15 +48,32 @@ type TlfHandle struct {
 	tlfID tlf.ID
 }
 
+// NewHandle returns a simple new Handle based on the given fields.
+// This is probably most useful for testing.
+func NewHandle(
+	ty tlf.Type,
+	resolvedWriters map[keybase1.UserOrTeamID]kbname.NormalizedUsername,
+	unresolvedWriters, unresolvedReaders []keybase1.SocialAssertion,
+	name tlf.CanonicalName, tlfID tlf.ID) *Handle {
+	return &Handle{
+		tlfType:           ty,
+		resolvedWriters:   resolvedWriters,
+		unresolvedWriters: unresolvedWriters,
+		unresolvedReaders: unresolvedReaders,
+		name:              name,
+		tlfID:             tlfID,
+	}
+}
```

Comment:
No `resolvedReaders` here. Is it because it's irrelevant now that we are all iteam based?

Reply:
Frankly it's because, in the old code, we didn't have any users of the constructor in tests that populated that field, so I didn't bother making a field for it.  If it bums you out, I'm happy to add it.

Reply:
No worries! I was mostly just curious. This looks good!